src/mscorlib/src/System/Globalization/DateTimeFormat.cs

```diff
@@ -959,8 +964,14 @@ internal static String Format(DateTime dateTime, String format, DateTimeFormatIn
             }
 
             if (format.Length == 1) {
-                if (format[0] == 'o' || format[0] == 'O') {
+                switch (format[0])
+                {
+                    case 'O':
+                    case 'o':
                         return FastFormatRoundtrip(dateTime, offset);
+                    case 'R':
+                    case 'r':
```

Comment:
Yeah, that is special handling for DateTimeOffset I need to apply. Do you know what InvalidFormatForLocal(...) is for and why it exists and is empty? MDA?